### ISSUE TITLE
.tell error: on fresh configs.

### ISSUE THREAD

**Issue by Serubin** [sopel-irc/sopel#865]
Freshly configured config on sopel 6.0.0
```
21:42 <@Serubin> .tell Laura testing
21:42 < Bot> AttributeError: 'Config' object has no attribute
                   'has_option' (file
"/usr/local/lib/python3.4/dist-packages/sopel/config/__init__.py", line 163, in
                   __getattr__)
```

**Comment by embolalia** [comment]
What exact version? There was an issue in 6.0.0a4 that is fixed in 6.0.0a5.

**Comment by embolalia** [comment]
Just noticed this is a duplicate of #857. Closing.
